cmd/loadTests.go

```diff
@@ -697,6 +713,7 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 				user = selectedUsers[threadIndex*numberOfUsers+userIndex-1]
 				username = user.Username
 				framework, err = tryNewFramework(username, user, 60*time.Minute)
+				StageCleanupSingle(user, framework)
```

Comment:
IMHO we should not delay startup of the test with some cleanup. If it is needed to run it before test, there need to be a different tool that will run at different stage of the pipeline.